src/App.js

```diff
@@ -1,13 +1,23 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Quote from './components/Quotes';
+import Home from './components/Home';
 
 class App extends React.Component {
   render() {
     return (
       <div>
-        <Calculator />
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/home" element={<Home />} />
+          <Route path="/calculator" element={<Calculator />} />
+          <Route path="/quotes" element={<Quote />} />
```

Comment:
- [ ] It would be better if you use only one route for the same component. Currently, you are using 2 routes for `Home`. Kindly fix this minor issue. 

```suggestion
          <Route path="/" element={<Home />} />
          <Route path="/calculator" element={<Calculator />} />
          <Route path="/quotes" element={<Quote />} />
```

I recommend you to use the `/` path so when the user starts the app can see the content.